### PR TITLE
chore(README): re-order list of Implementations to be alphabetical: 

### PR DESCRIPTION
**Pull request by nelsonic**
As discussed in: https://github.com/multiformats/multihash/pull/154#issuecomment-1332412681 💬 
This PR alphabetises the list of implementations: ⬇️ 

## Before:

image

## After: 

image

Didn't change any of the text of the lines, just a copy-paste to reduce PR review time. 👍 
But in a future update the links could be made more consistent. 💭 
Thanks again for maintaining this awesome resource! ❤️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,7 +131,7 @@ They disagree. :(
 
 ## Disclaimers
 
-Warning: **obviously multihash values bias the first two bytes**. Do not expect them to be uniformly distributed. The entropy size is `len(multihash) - 2`. Skip the first two bytes when using them with bloom filters, etc. Why not _ap_pend instead of _pre_pend? Because when reading a stream of hashes, you can know the length of the whole value, and allocate the right amount of memory, skip it, or discard it.
+Warning: **obviously multihash values bias the first two bytes**. Do not expect them to be uniformly distributed. The entropy size is `len(multihash) - 2`. Skip the first two bytes when using them with bloom filters, etc. Why not \_ap_pend instead of \_pre_pend? Because when reading a stream of hashes, you can know the length of the whole value, and allocate the right amount of memory, skip it, or discard it.
```
Comment (nelsonic) [inline README.md:134]: Sorry about this auto-format update. Going to revert it now. ⏳

Comment (nelsonic) [reply README.md]: Reverted.

**Comment by nelsonic** [comment]
@vmx thanks again. 🙏

**Review by vmx — APPROVED** [review]
Thanks for helping making things cleaner, that looks good.

**Pull request merged**